MapboxCoreNavigation/TunnelIntersectionManager.swift

```diff
@@ -98,7 +109,7 @@ open class TunnelIntersectionManager: NSObject {
         isAnimationEnabled = true
     }
     
-    @objc public func suspendTunnelAnimation(for manager: CLLocationManager, at location: CLLocation, routeController: RouteController) {
```

Comment:
@vincethecoder noting, I removed the manager from many functions here as they were unused.